src/main.rs

```diff
@@ -25,14 +24,32 @@ enum SubCommand {
     Teardown(teardown::Teardown),
 }
 
-fn main() -> Result<(), Box<dyn Error>> {
+fn main() {
     env_logger::init();
     let opts = Opts::parse();
 
     let file = opts.file.unwrap_or_else(|| String::from("/dev/stdin"));
     match opts.subcmd {
-        SubCommand::Setup(setup) => setup.exec(file),
-        SubCommand::Teardown(teardown) => teardown.exec(file),
+        SubCommand::Setup(setup) => {
+            if let Err(err) = setup.exec(file) {
+                let er = netavark::error::NetavarkError {
+                    error: format!("{}", err),
+                    errno: 1,
+                };
+                println!("{}", er.to_string());
+                quit::with_code(er.errno);
+            }
+        }
+        SubCommand::Teardown(teardown) => {
+            if let Err(err) = teardown.exec(file) {
+                let er = netavark::error::NetavarkError {
+                    error: format!("{}", err),
+                    errno: 1,
+                };
+                println!("{}", er.to_string());
+                quit::with_code(er.errno);
```

Comment:
Is there any point in using a new dependency for this. Can you please just use `std::process::exit()` directly.

Reply:
sure thing, just that I saw a few recommendations online for quit. I'll change this in a bit. Thanks

Reply:
done, ptal